ui/app/AppLayouts/Chat/controls/community/HoldingsDropdown.qml

```diff
@@ -15,35 +15,55 @@ import utils 1.0
 StatusDropdown {
     id: root
 
-    property real tokenAmountValue: 0
+    property var store
+
+    property var itemKey
+    property real tokenAmount: 0
     property string tokenName: d.defaultTokenNameText
     property url tokenImage: ""
+    property real collectibleAmount: 1
+    property string collectibleName: d.defaultCollectibleNameText
+    property url collectibleImage: ""
     property int operator: SQ.Utils.Operators.None
     property bool withOperatorSelector: true
 
-    signal addToken(string tokenText, url tokenImage, int operator)
+    signal addItem(var itemKey, string itemText, url itemImage, int operator)
 
     function reset() {
-        root.tokenAmountValue = 0
+        d.currentTabIndex = 0
+        root.itemKey = undefined
+        root.tokenAmount = 0
         root.tokenName = d.defaultTokenNameText
         root.tokenImage = ""
+        root.collectibleAmount = 1
+        root.collectibleName = d.defaultTokenNameText
+        root.collectibleImage = ""
         root.operator = SQ.Utils.Operators.None
     }
 
     QtObject {
         id: d
 
-        readonly property bool ready: root.tokenAmountValue > 0 && root.tokenName !== d.defaultTokenNameText
+        // Internal management properties:
+        readonly property bool tokensReady: root.tokenAmount > 0 && root.tokenName !== d.defaultTokenNameText
+        readonly property bool collectiblesReady: root.collectibleAmount > 0 && root.collectibleName !== d.defaultCollectibleNameText
+        readonly property string tokensState: "TOKENS"
+        readonly property string collectiblesState: "COLLECTIBLES"
+        readonly property string ensState: "ENS"
+        property bool isTokensLayout: true
+        property int currentTabIndex: 0
 
         // By design values:
         readonly property int initialHeight: 232
         readonly property int mainHeight: 256
+        readonly property int mainExtendedHeight: 276
         readonly property int operatorsHeight: 96
         readonly property int extendedHeight: 417
         readonly property int defaultWidth: 289
         readonly property int operatorsWidth: 159
 
         property string defaultTokenNameText: qsTr("Choose token")
+        property string defaultCollectibleNameText: qsTr("Choose collectible")
```

Comment:
When token is selected and added we end up with "Choose token" in both tabs.

Reply:
Done! Please check it again!